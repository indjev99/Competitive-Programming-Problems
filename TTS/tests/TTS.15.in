100 10
8689.6 -4948.83
7182.31 6958.04
-1372.34 -9905.39
-9041.83 4271.46
-9426.04 -3339.14
8686.06 -4955.04
9981.31 -611.041
-7839.58 6208.14
-8987.79 4384.02
-9909.13 1345.07
-5578.55 8299.38
-9686.67 2483.64
1426.76 9897.7
6216.97 -7832.57
8592.83 -5115.01
-1334.78 -9910.52
-6790.97 -7340.48
-767.509 9970.5
-9994.48 -332.099
-5255.61 -8507.56
5541.41 -8324.23
7292.6 -6842.37
9460.97 -3238.82
2157.92 9764.39
187.53 -9998.24
-3810.69 9245.46
9000.47 4357.93
-8400.49 -5425.11
-1938.47 -9810.32
-6206.4 -7840.96
-3446.81 9387.2
-7399.09 -6727.07
9929.88 -1182.14
6700.18 7423.45
8020.09 -5973.12
81.8588 -9999.66
-3389.48 -9408.05
-9805.89 -1960.76
850.426 -9963.77
-2571.88 -9663.61
-8388.91 -5442.99
9915.39 1298.09
9654.7 -2605.14
-2030.49 -9791.69
-832.264 9965.31
2001.6 -9797.63
1580.12 -9874.37
-7988.34 6015.51
8962.14 4436.22
3876.9 9217.9
-9815.07 1914.26
-786.97 9968.99
-3626.32 -9319.32
-7624.52 -6470.45
-7985.67 -6019.06
5967.23 -8024.47
-157.243 9998.76
7443.92 -6677.43
9929.37 1186.4
-9932.29 1161.71
6102.97 7921.73
-5425.06 8400.52
-9289.08 3703.12
-7069.93 -7072.2
-8739.85 4859.53
-9818.48 -1896.71
-3800.35 -9249.72
8656.67 -5006.21
4252.35 -9050.83
-9932.52 -1159.8
-9748.54 -2228.43
1866.32 -9824.3
-9525.55 3043.68
5180.53 -8553.48
4452.98 8953.82
-4122.86 -9110.54
9356.68 3528.82
9587.72 2841.75
-4421.86 8969.23
9962.3 867.532
9240.18 -3823.5
9001.22 -4356.38
9025.11 -4306.67
-8422.21 -5391.32
-9892.73 1460.77
-3324.03 9431.37
-9696.97 -2443.1
2133.13 -9769.84
-3550.08 9348.63
-9845.19 1752.81
-580.98 9983.11
-1592.24 9872.43
8207.34 5713.1
-1686.65 9856.73
-6123.96 7905.51
-9129.65 4080.37
9559.56 -2935.11
3122.1 -9500.13
-9871.84 1595.86
-4151.9 9097.35
-7108.89 5224.2
-304.83 3575.01
-954.09 -4496.42
7780.89 7329.35
-5554.15 2627.66
3305.73 4116.39
4190.95 9576.66
4041.58 -9172.97
-5941.53 1640.01
-1521.31 2411.96
