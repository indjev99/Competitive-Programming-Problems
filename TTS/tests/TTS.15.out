32508.872215
